### ISSUE TITLE
Accessibility - Information Screens - Missing Heading

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#22]
All the screens under the 'More Information' section are missing a heading at the top of the page and not read as such by TalkBack.

**Comment by aquilent-dgemoets** [comment]
See discussion for #20.

**Comment by jfc3-dol** [comment]
I reviewed the ILAB 'Sweat and Toil' application on OPA's 'Galaxy Tab S' and an ILAB employees 'LG Tribute 2' and both did not read any headings. 

From my understanding two of the 'More information' screens (About the App and Methodology) where HTML pages and had the headings in them. 

I thought the OSHA 'Heat Index' application had similar screens and the headings with their proper level were read for HTML screens before. The other Android screens that had what equated to headings were read too.

**Comment by aquilent-dgemoets** [comment]
I checked the repository for the (unpublished) Android version of the OSHA 'Heat Index' application and didn't see any added accessibility code/configuration.  Could you confirm that the behavior you are remembering was for the Android version of the app?

**Comment by jfc3-dol** [comment]
None of the HTML based views for "About This App" and "Methodology" are having any of their headings read.

For the rest of the screen none of the PDF's are being read at all as of now.

**Comment by jfc3-dol** [comment]
The "About This App" screen is now reading heading level one (h1) and heading level two (h2) items on the page. The "Methodology" screen is only reading the first item as a heading level one (h1) and not the other items below as heading level two (h2) items as you would expect.

**Comment by jfc3-dol** [comment]
The following is still true:

The "About This App" screen is now reading heading level one (h1) and heading level two (h2) items on the page.

The "Methodology" screen is only reading the first item as a heading level one (h1) and not the other items below as heading level two (h2) items as you would expect. They are being read as links. Not sure if them being links if over riding that they are supposed to heading level two (h2) items.

**Comment by jfc3-dol** [comment]
By creating the links in the paragraphs and taking them off the heading level two (h2) text they are now read as heading level two (h2).

Closing issue since it's corrected.
